### ISSUE TITLE
Process has no package identity

### ISSUE THREAD

**Issue by CameronBieganek** [microsoft/terminal#14863]
### Windows Terminal version

1.17.10234.0

### Windows build number

10.0.19044.2486

### Steps to reproduce

I am working on a locked down corporate laptop with Windows 10. I had Terminal installed via one of the binaries from Github, but then after an OS update, it stopped working. By "stopped working", I mean that I couldn't launch Terminal at all, and there were no error messages when I tried to launch Terminal. Just nothing would happen at all when I tried to launch it.

When I tried to install the new 1.17 binary, I get the error shown below. I get the same thing when I try to reinstall 1.16.

Steps to reproduce:
- Download the msixbundle file for 1.17.
- Attempt to install Terminal via msixbundle.
- Hit the "Run" button in Microsoft Store after installation.

### Expected Behavior

Terminal launches.

### Actual Behavior

![terminal](https://user-images.githubusercontent.com/8310743/219438166-4afdd538-230f-4410-8266-2afa87fbe8eb.png)

**Comment by zadjii-msft** [comment]
Hmm, we've seen this before (#11760) in an environment where the OP was using "Admin by Request" - any chance that applies here/?

**Comment by CameronBieganek** [comment]
As far as I know we are not using Admin By Request at my company, but I don't know for sure...

**Comment by dhoehna** [comment]
@DHowett-MSFT Seems like, maybe, a terminal issue?  This ring a bell?

**Comment by DHowett** [comment]
@dhoehna I cannot see how! We aren't producing this dialog box, and it is before a single instruction in any of our executables is run.

**Comment by dhoehna** [comment]
@DHowett Yeah.  My mistake.  After looking at this a second time I also came to the same conclusion.  Sorry.
